virttest/utils_misc.py

```diff
@@ -2565,11 +2565,12 @@ def __get_unit_index(M):
     from_index = __get_unit_index(unit)
     to_index = __get_unit_index(order_magnitude)
     scale = int(factor) ** (to_index - from_index)
-    if scale > 0:
-        data_size = float(value) / abs(scale)
+    data_size = float(value) / scale
+    # Control precision to avoid scientific notaion
+    if data_size.is_integer():
```

Comment:
>I think return "%.20f" % data_size is enough.

Not really, %.20 will keep the precision to .20. 
I will use %.1f for integer to return the consistent result/